### PR TITLE
Alex/add addin common strings

### PR DESCRIPTION
**Pull request by alexgomezlf**
Add strings to be used in the Outlook Addin, that can be used in other places

### CODE REVIEW

File: resources/laserfiche-base/pt-BR.json [hunk 1]
```diff
@@ -59,6 +115,8 @@
     "TOKEN_PARENT_FRIENDLYNAME": "Campo Pai",
     "TOKEN_TIME_FRIENDLYNAME": "Hora",
     "TOKEN_USERNAME_FRIENDLYNAME": "Nome do usuário",
+    "UNABLE_TO_LOAD_ALL_FIELDS": "Incapaz de carregar todos os campos: ",
+    "UNABLE_TO_RETRIEVE_REPO_INFO": "Não é possível recuperar as informações do repositório para '{0}' de {1}.{2}",
```
Comment (alexgomezlf) [inline resources/laserfiche-base/pt-BR.json]: This looks weird as a translation for `Unable to retrieve repository information: `. Should I change it back to English and translate it again the next time we do translations? Of should I change it to `Não é possível recuperar as informações do repositório: `
@paolo-arge-lf @Ke-Jin-Laserfiche

File: resources/laserfiche-base/ar.json [hunk 2]
```diff
@@ -23,25 +47,57 @@
...
+    "REMOVE_REPOSITORY_TITLE": "هل تريد إزالة المستودع؟",
+    "REPOSITORY": "مستودع",
+    "REPOSITORY_ID": "رقم تعريف الإيداع",
+    "REPOSITORY_NAME": "اسم المستودع",
     "REQUIRED": "مطلوب",
+    "SAVE_CHANGES": "حفظ التغييرات",
+    "SAVE_TO_LASERFICHE": "حفظ في Laserfiche",
+    "SAVED_FILE_TO_LASERFICHE": "تم الحفظ في Laserfiche:",
+    "SAVED_FILE_TO_LASERFICHE_MISSING": "تم الحفظ ببيانات تعريف مفقودة:",
+    "SAVED_TO_LASERFICHE": "تم الحفظ في Laserfiche\r\n",
```
Comment (Ke-Jin-Laserfiche) [inline resources/laserfiche-base/ar.json]: is the new line character necessary?

Comment (alexgomezlf) [reply resources/laserfiche-base/ar.json]: removed

File: resources/laserfiche-base/es.json [hunk 3]
```diff
@@ -1,64 +1,117 @@
...
+    "NO_ADDITIONAL_FIELDS_ASSIGNED": "No hay campos adicionales asignados",
+    "NO_FOLDER_SELECTED": "No se seleccionó carpeta.",
+    "NO_MATCHING_ENTRIES_FOUND": "No se han encontrado coincidencias",
+    "NO_MATCHING_FIELDS_FOUND": "No se encontraron campos que coincidan.",
     "NO_TEMPLATE_ASSIGNED": "No hay plantilla asignada",
-    "NUMBER_FIELD_VALIDATOR_INVALID_MESSAGE": "El campo de números debe ser un número válido",
+    "NOT_AVAILABLE_WITH_TOKENS": "No está disponible con un tokens",
+    "NUMBER": "Número",
+    "NUMBER_FIELD_MUST_BE_VALID_NUMBER": "El campo de números debe ser un número válido",
     "OK": "Aceptar",
```
Comment (Ke-Jin-Laserfiche) [inline resources/laserfiche-base/es.json:70]: this is "Accept" or "Ok"?

File: resources/laserfiche-base/pt-BR.json [hunk 4]
```diff
@@ -1,64 +1,116 @@
...
+    "COMMENT": "Comentário",
+    "CONFIGURE_REPOSITORY_SETTINGS": "Definir as configurações do repositório",
+    "COULD_NOT_SAVE_0": "Não foi possível salvar  \"{{filename}}\".",
+    "COULD_NOT_SAVE_0_FILE_TYPE_NOT_SUPPORTED": "Não foi possível salvar \"{{filename}}\". Tipo de arquivo não compatível.",
+    "DATA": "Dados",
+    "DATE": "Data",
+    "DATE_FIELDS_MUST_BE_IN_FORMAT_0": "Os campos de data devem estar em formato {0}.",
+    "DATE_PROVIDED_IS_INVALID": "A data fornecida é inválida.",
+    "DATE_TIME": "Data/Tempo",
+    "DATE_TIME_FIELDS_MUST_BE_IN_FORMAT_0": "Os campos de data/hora devem estar no formato {0}.",
```
Comment (Ke-Jin-Laserfiche) [inline resources/laserfiche-base/pt-BR.json:29]: missing an entry "TIME_FIELDS_MUST_BE_IN_FORMAT_0": "Os campos de hora devem estar no formato {0}."

Comment (alexgomezlf) [reply resources/laserfiche-base/pt-BR.json:29]: nice catch

File: resources/laserfiche-base/es.json (hunk 3 above)
Comment (alexgomezlf) [reply resources/laserfiche-base/es.json:70]: It means "Accept" as the literal translation but I think "Aceptar" is what they would use for affirmative buttons..

**Review by Ke-Jin-Laserfiche — APPROVED** [review]

**Pull request merged**